src/test/java/org/takes/misc/VerboseIterableTest.java

```diff
@@ -0,0 +1,38 @@
+package org.takes.misc;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.collection.IsIterableWithSize;
+import org.junit.Test;
+
+/**
+ * Tests for {@link VerboseIterable}.
+ * 
+ * @author marcus.sanchez (sanchez.marcus@gmail.com)
+ * @version $Id$
+ * @since 0.15.1
+ */
+public class VerboseIterableTest {
+
+    /**
+     * VerboseIterable can return correct size collection.
+     */
+    @Test
+    public void returnsCorrectSize() {
+        final List<String> VALID_LIST = Arrays.asList(
```

Comment:
@m4solutions uppercase names are reserved for static variables. non-static variables should be lowercase and 1-word (avoid compound names). see this for more info http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html
I suggest `valid`.